msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -1,166 +1,88 @@
 ---
-title: Enable resource-specific consent in Teams
-description: Learn about supported granular Resource-specific consent (RSC) permissions which allows team owners and chat owners to grant consent for an application.
+title: Grant RSC permissions to an app
```

Comment:
The file should be renamed to grant-resource-specific-consent.md now.

Reply:
Renamed .md file as suggested.